Production/govuk_ios/Resources/Assets.xcassets/OnboardingImages/onboarding_placeholder_screen_1.imageset/Contents.json

```diff
@@ -1,9 +1,8 @@
 {
   "images" : [
     {
-      "filename" : "onboarding_placeholder_screen_1.png",
-      "idiom" : "universal",
-      "scale" : "1x"
+      "filename" : "On the go - light mode.svg",
```

Comment:
Doesn't appear we're following a naming convention for image files in the project as I can see `homeLogo.png` and `recent_activity_cell_chevron.pdf`? 

I have just dropped images in as they were already named so let me know if this needs changing?